R/ospsuite-env.R

```diff
@@ -0,0 +1,3 @@
+# Environment that holds various global variables and settings for the ospsuite,
+# It is not exported and should not be directly manipulated by other packages.
+ospsuiteEnv <- new.env(parent = emptyenv())

```

Comment:
Why having a separate file, and not just initialize in `init.package.R`?

Reply:
This file can document all the stuff we are going to put in there.  This can become quite long and that's why I followed ggplot structure and created a separate file

Reply:
OK.